modules/meteroid/crates/meteroid-repository/refinery/migrations/V0025__invoice_add_amount.sql

```diff
@@ -0,0 +1,3 @@
+-- Modify "customer" table
+ALTER TABLE "invoice"
+    ADD COLUMN "amount_cents" INTEGER;
```

Comment:
`integer` is restricted to have a max value of ~21.4 million EUR/USD/etc. 
Do we want to have such restrictions? I would make it bigint and do the same for respective rust type.